test/parallel/test-runner-run.mjs

```diff
@@ -101,4 +101,20 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     assert.strictEqual(result[11], '# todo 0\n');
     assert.match(result[12], /# duration_ms \d+\.?\d*/);
   });
+
+  it('should skip tests not matching testNamePatterns - RegExp', async () => {
+    const result = await run({ files: [join(testFixtures, 'test/skip_by_name.cjs')], testNamePatterns: [/executed/] })
+      .compose(tap)
```

Comment:
```suggestion
```
NIT: tap is the default

Reply:
when removing the `compose` it returns the stream items without the `tap` reporter changing the data:
![image](https://user-images.githubusercontent.com/67366097/233421133-ebc1bfe5-ac28-4fc0-9fc6-b814f6d5fdb0.png)